YAML/bash_test_bot.yaml

```diff
@@ -0,0 +1,45 @@
+ ## ID - auto generated or manual alpha numeric, must be unique across RLC
+id: bash_test_bot
+
+## Name - human readable name (e.g., Install MongoDB, Change JIRA password...)
+name: bash test bot
+
+## Desc - Few line brief about the BOT and its characteristics.
+desc: shows only messages for test purpose.
+
+## Category -> Check, Run, Meta, Predict...
+functionality: Service Management
```

Comment:
The functionality of this BOT should not be "Service Management". Give the functionality as "Sample"